### PR TITLE
Fix selenium depreciation warnings

### PR DESCRIPTION
**Pull request by ruzv**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -145,9 +160,13 @@ The class "Elements" implements default locator methods that are part of the App
         return Elements(driver, locator_type, locator)
 ```
 
-Once you create the definition of the element with the static method e(driver, l_type, locator) you can start using a series of action/assertion methods that are built-in within Elements class.
+Once you create the definition of the element with the static method e(driver,
+l_type, locator) you can start using a series of action/assertion methods that
+are built-in within Elements class.
 
-Action methods are the ones meant to do something over the UI. In case the cannot be performed, they will rise an error which will show all the information needed for debugging:
+Action methods are the ones meant to do something over the UI. In case the
```
Comment (theRealAlpaca) [inline README.md:167]:
```suggestion
Action methods are the ones meant to do something over the UI. In case they
```

File: README.md [hunk 2]
```diff
@@ -145,9 +160,13 @@ The class "Elements" implements default locator methods that are part of the App
...
+cannot be performed, they will rise an error which will show all the information
```
Comment (theRealAlpaca) [inline README.md:168]:
```suggestion
cannot be performed, an error will be raised that will show all the information
```

File: testui/elements/testui_element.py [hunk 3]
```diff
@@ -1,3 +1,4 @@
+from __future__ import annotations
```
Comment (theRealAlpaca) [inline testui/elements/testui_element.py:1]: Why is this needed?

File: testui/support/appium_driver.py [hunk 4]
```diff
@@ -297,25 +302,36 @@ def start_selenium_driver(
...
-    firefox_options=None,
-) -> WebDriver:
-    options = chrome_options
-    if firefox_options is not None:
-        options = firefox_options
+    browser: str or None = None,
+    # chrome_options: ChromeOptions or None = None,
+    # firefox_options=None,
+    options: ChromeOptions or FirefoxOptions or None = None,
+) -> RemoteWebDriver:
```
Comment (theRealAlpaca) [inline testui/support/appium_driver.py:309]: This is a breaking change as well. Maybe create an overload method?

File: testui/support/appium_driver.py [hunk 5]
```diff
@@ -297,25 +302,36 @@ def start_selenium_driver(
...
+        logger.log(f"setting options: {options.to_capabilities().__str__()}")
 
-    logger.log("setting capabilities: " + desired_caps.__str__())
+    # options = chrome_options
+    # if firefox_options is not None:
+    #     options = firefox_options
+
+    # if options is not None:
+    #     logger.log(f"setting options: {options.to_capabilities().__str__()}")
+
```
Comment (theRealAlpaca) [inline testui/support/appium_driver.py:321]: Is this needed?

File: testui/support/testui_driver.py [hunk 6]
```diff
@@ -6,11 +6,11 @@
 from pathlib import Path
 
 from appium.webdriver.common.touch_action import TouchAction
-from appium.webdriver.webdriver import WebDriver
 from selenium.webdriver import TouchActions, ActionChains
 from selenium.common.exceptions import WebDriverException
+from selenium.webdriver.remote.webdriver import WebDriver as RemoteWebDriver
```
Comment (theRealAlpaca) [inline testui/support/testui_driver.py:11]: Why do you create an alias instead of importing it as `WebDriver`?

File: testui/support/appium_driver.py [hunk 7]
```diff
@@ -10,6 +11,9 @@
 from ppadb.client import Client as AdbClient
 from appium.webdriver import Remote
 from appium.webdriver.webdriver import WebDriver
+from selenium.webdriver.chrome.options import Options as ChromeOptions
+from selenium.webdriver.firefox.options import Options as FirefoxOptions
+from selenium.webdriver.remote.webdriver import WebDriver as RemoteWebDriver
```
Comment (theRealAlpaca) [inline testui/support/appium_driver.py:16]: Is this alias really necessary?

**Review by theRealAlpaca — CHANGES_REQUESTED** [review]

File: testui/support/appium_driver.py (hunk 5 above)
Comment (ruzv) [reply testui/support/appium_driver.py:321]: to annotate methods that return self.

File: testui/elements/testui_element.py (hunk 3 above)
Comment (ruzv) [reply testui/elements/testui_element.py:1]: to annotate methods that return self.

File: testui/support/testui_driver.py (hunk 6 above)
Comment (ruzv) [reply testui/support/testui_driver.py]: wanted to add some clarity, but rn it is impossible to do that. so this change is unnecessary.

File: testui/elements/testui_element.py (hunk 3 above)
Comment (theRealAlpaca) [reply testui/elements/testui_element.py:1]: I think you can write them like this and not use this `__future__` import:
```py
def clear(self) -> "Elements":
```

**Review by theRealAlpaca — CHANGES_REQUESTED** [review]

**Review by paulsgrudups — APPROVED** [review]

**Review by theRealAlpaca — APPROVED** [review]
